src/stm32/Makefile

```diff
@@ -47,6 +49,10 @@ src-$(CONFIG_MACH_STM32F103) += ../lib/stm32f1/system_stm32f1xx.c
 src-$(CONFIG_MACH_STM32F103) += stm32/adc.c
 src-$(CONFIG_MACH_N32G45x) += ../lib/stm32f1/system_stm32f1xx.c
 src-$(CONFIG_MACH_N32G45x) += ../lib/n32g45x/n32g45x_adc.c stm32/n32g45x_adc.c
+src-$(CONFIG_MACH_AT32F415) += ../lib/stm32f1/system_stm32f1xx.c
```

Comment:
If you're already setting the CONFIG_MACH_ST32F1 flag, are these required still?